bin/test-dir

```diff
@@ -4,4 +4,8 @@ set -e
 
 source bin/require_fd
 
-$fd '\.(hack|php)$' "$@" | xargs -r -n 256 bin/ts-errors
+# Note: By default, fd ignores hidden directories,
+#       hidden files, and .gitignore patterns
+# To change this behavior, use --hidden and/or --no-ignore with fd calls
+
+{ fd . -e php "$@" | xargs -r egrep -l "^<\?hh" & fd . -e hack "$@"; } | xargs -r -n 256 bin/ts-errors
```

Comment:
We might consider expanding this at some point to some other extensions that seemed to be allowed (although not really used by us).

https://github.com/facebook/hhvm/blob/6fb8cdcf2c397f724832f31315bc586bfb5ac951/hphp/hack/src/utils/findUtils.ml#L16-L32